src/main/java/com/work/management/service/log/impl/LoggingServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.work.management.service.log.impl;
+
+import com.work.management.service.log.LoggingService;
+import java.util.Collection;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LoggingServiceImpl implements LoggingService {
+
+  private static final Logger logger = LoggerFactory.getLogger(LoggingServiceImpl.class);
+
+  public void logRequest(HttpServletRequest httpServletRequest, Object body) {
+    StringBuilder stringBuilder = new StringBuilder();
+    Map<String, String> parameters = buildParametersMap(httpServletRequest);
+
+    stringBuilder.append("REQUEST ");
```

Comment:
this could be reWritten as stringBuilder.append("REQUEST ")
.append(...)
.append(....)
.append(....)

chain all the calls instead of using stringBuilder again and again.